apps/backend/src/app/api/latest/cron/send-email-digest/route.ts

```diff
@@ -0,0 +1,92 @@
+import { getEmailConfig, sendEmail, sendEmailFromTemplate } from "@/lib/emails";
+import { getSoleTenancyFromProject } from "@/lib/tenancies";
+import { prismaClient } from "@/prisma-client";
+import { Project } from "@prisma/client";
+import { getEnvVariable } from "@stackframe/stack-shared/dist/utils/env";
+
+export async function GET(request: Request) {
+  const authHeader = request.headers.get('authorization');
+  if (authHeader !== `Bearer ${getEnvVariable('CRON_SECRET')}`) {
+    return new Response('Unauthorized', {
+      status: 401,
+    });
+  }
+
+
+  // get projects from tenancy IDs
+  const emails = await prismaClient.sentEmail.findMany({
+    where: {
+      createdAt: {
+        gte: new Date(Date.now() - 1000 * 60 * 60 * 24),
+      },
+    },
+    include: {
+      tenancy: {
+        include: {
+          project: true,
+        }
+      },
+    }
+  });
+
+  const projectWithEmails: Record<string, { project: Project, emails: (typeof emails[number])[] } | undefined>
```

Comment:
never use objects like maps, instead make this a `Map<A, B>`. otherwise it's susceptible to prototype pollution

Reply:
Done